src/ManageCourses.Api/Controllers/DataController.cs

```diff
@@ -172,6 +176,48 @@ private void ProcessPayload(Payload payload)
                     }
                 );
             }
+
```

Comment:
As per my comment on the importer, I think this is likely to need to become a separate operation before long. Especially as it's currently combined with dropping the existing data. Happy for this to go in and be iterated on.